src/array_api_stubs/_draft/array_object.py

```diff
@@ -359,7 +376,7 @@ def __dlpack_device__(self: array, /) -> Tuple[Enum, int]:
 
         Returns
         -------
-        device: Tuple[Enum, int]
+        device: Tuple[:class:`~enum.Enum`, int]
```

Comment:
I assume this is an artifact from your code editor and can be reverted?

Reply:
I'll revert this and hopefully the docs still compile correctly. This was just a sphinx thing I tried.
I'm also planning to squash my commits, unless y'all use Squash and Merge.

Reply:
Done!